q2_composition/_ancombc.py

```diff
@@ -120,6 +120,25 @@ def _ancombc(table, metadata, formula, p_adj_method, prv_cut, lib_cut,
                                  ' \n\n'
                                  ' column::value pair with a value that was'
                                  ' not found: "%s"' % i)
+
+            # check that IDs associated with chosen reference level(s) are
+            # present within the input table
+            level_value_idx = meta.index[meta[column] == level_value].tolist()
+            table_idx_list = table.index.tolist()
+
+            for id in level_value_idx:
+                if id not in table_idx_list:
+                    raise ValueError('Value provided in `reference_levels`'
+                                     ' parameter not associated with any IDs'
+                                     ' in the feature table. Please make sure'
+                                     ' the value(s) selected in each'
+                                     ' column::value pair are associated with'
+                                     ' IDs present in the feature table.'
+                                     ' \n\n'
+                                     ' Value not associated with any IDs in'
+                                     ' the table: "%s"' % level_value,
+                                     ' ID not found in table: "%s"' % id)
```

Comment:
Not sure if this message makes the most sense, or if there's a better way to word this.

Reply:
I think this is a good error message.